packages/core/custom-elements-lsp/README.md

```diff
@@ -129,6 +137,19 @@ You just need to setup VSCode to use your local typescript install as by default
 
 If you have an LSP setup for typescript this should work straight away using the project's TypeScript.
 
+### JetBrains
+
+This section covers all of JetBrains IDEs, such as WebStorm and IntelliJ.
+[Currently](https://youtrack.jetbrains.com/issue/WEB-62815/Ability-to-use-tsserver-to-implement-all-LSP-functionality-from-TypeScript) there is only partial support for the CEP.
+* Full diagnostics support.
+* Partial completions support, provided by the IDE itself *not* using the CEP - but enhanced because of the manifest files that are used with the plugin.
+* No support for quicklook/quickinfo, as JetBrains run this closed source without using `tsserver` in their IDEs.
+
+To use the LSP in your JetBrains IDE.
+1. Launch the IDE with the project at the root of the monorepo.
+2. Open the preferences menu option from the settings.
+3. Navigate to the `Typescript` settings in the `Languages & Frameworks` settings, and ensure that the typescript option is set to the `node_modules/typescript` of your _local_ project, as shown in the image. This may be the default already, in which case you don't need to do anything. You'll also want to enable at least the three options which are enabled in the image below.
```

Comment:
> which are enabled in the image below.

what image? do you mean above?

Reply:
Thanks, addressed in https://github.com/genesiscommunitysuccess/custom-elements-lsp/pull/80/commits/593fcd3341c369e5a8294c9852cff9cca1863480